components/TestimoniesTable/TestimoniesTable.js

```diff
@@ -17,7 +18,9 @@ const TestimonyRow = ({ testimony }) => {
 
   return (
     <tr>
-      <td>{testimony.billId}</td>
+      <td>
+        <Link href={`/bill?id=${testimony.billId}`}>{testimony.billId}</Link>
```

Comment:
Let's also wrap the text in formatBillId: https://github.com/codeforboston/advocacy-maps/blob/559d28aab969cf61c4ab6074a14e1a0fd9dd8ffe/components/ViewBills/ViewBills.js#L66

Reply:
Sounds good.  Added the requested changes.